### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by vinielk**
Fixed home page and navbar. Need to merge in other pull requests to complete other pages.

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -1,22 +1,12 @@
...
  */
 
 require get_template_directory() . '/inc/cleanup.php';
 require get_template_directory() . '/inc/enqueue.php';
 require get_template_directory() . '/inc/theme-support.php';
-
-require get_template_directory() . '/inc/title-cf.php';
-require get_template_directory() . '/inc/paragraph-cf.php';
-require get_template_directory() . '/inc/highlightedSection-cf.php';
-require get_template_directory() . '/inc/content-right-side-img-cf.php';
```
Comment (pritpalc) [inline functions.php]: Are you working with the most up to date code? I already removed this code and the custom field php files.

Comment (vinielk) [reply functions.php]: Master branch still has those lines which is why they’re showing up. I can double check tonight

Comment (pritpalc) [reply functions.php]: I know but my branch `refactor` shouldn't have any of those. My local copy doesn't have it and I have nothing else to commit.

Comment (pritpalc) [reply functions.php]: Nvm stupid questions, i figured it out

**Pull request merged**